### ISSUE TITLE
Making HEI Erasmus Code or PIC mandatory in HEI definition

### ISSUE THREAD

**Issue by sascoms** [erasmus-without-paper/ewp-specs-api-discovery#13]
Dear friends,

We propose to make both Erasmus Code and PIC information mandatory in the HEI definition block of the manifest file.

REF: https://github.com/erasmus-without-paper/ewp-specs-api-discovery/blob/0d8130f36df6231dab828138a9e15af3a4b53131/manifest.xsd#L128

This is needed to identify and find the HEI from the Erasmus charter list. As Erasmus charter list does not contain the SCHAC values of the HEIs, we need to match a HEI with a common property that exists in both lists. And this is either Erasmus code or PIC (HEI label/name is not applicable as name of the HEI differs even between MT and Charter list).

Currently, when checking manifest files and the registry catalogue, we have seen that there are many HEIs with missing Erasmus code and PIC numbers.

**Comment by janinamincer-daszkiewicz** [comment]
Personally I wouldn't mind. I hope more service providers will support this idea. The problems I see are however the following:
- PIC are obsolete, replaced by the Organizational code (OID), which is however not commonly recognized.
- Eventually we may want to have services in the network from the institutions which do not possess Erasmus Code.
MAy be should just make them highly recommended.

**Comment by sascoms** [comment]
PIC or Erasmus or any other ID would work. 

Taking OID into consideration, some questions are as follows:
- Does every HEI know its OID?
- Is it listed in MT HEI database?
- Is it included as a column in Erasmus charter list?
- Where HEIs can find out their OIDs?

**Comment by janinamincer-daszkiewicz** [comment]
The only one every HEI has and should know and we can make it mandatory is SCHAC ;) 
The others are a very useful add-on.

**Comment by sascoms** [comment]
The problem with SCHAC is unfortunately it is not listed in Erasmus Charter List. And I do not think it exists in MT db, too.
As SCHAC is decided by the HEIs themselves.

For now, making at least one ID info (Erasmus Code or OID or PIC) would be great.

Thanks for the ideas and comments.

**Comment by janinamincer-daszkiewicz** [comment]
Generally I agree, but if there is a choice they can not be mandatory. Highly recommended - yes.

**Comment by georgschermann** [comment]
Not all participating HEIs have a PIC or Erasmus Code, these are needed for auto discovery, but making one of them mandatory would lead to some HEIs be excluded from the network or fill in non existing Erasmus codes (which is also already the case)

**Comment by umesh-qs** [comment]
I agree with @georgschermann. At best we can change the description to say that if partner is aware of any of these IDs then they must pass it.

**Comment by georgschermann** [comment]
another thing, which was brought to our attention in this regard is, that some university use different providers across faculties, so it may be beneficial for e.g. the faculties to leave out the erasmaus code so that there are not multipe schacs per erasmus code, which may cause problems during schac lookup
